pydsa/dfs.py

```diff
@@ -0,0 +1,16 @@
+# Depth First Search
+# Complexity: O(V+E)
+#uses concept of list
+
+def dfsearch(graph,s):
```

Comment:
> definitely, function name must be DFS but then it doesn't work. import module error arise which i discussed. 

If it doesn't work then probably you are importing the wrong path. See how `bfs` works.